react-client/src/pages/dashboard/dashboard.js

```diff
@@ -232,59 +248,55 @@ function Dashboard() {
                               {truncateText(dmp.title, 50)}
                             </Link>
 
-                            {/*
-                        <a
-                          href="#"
-                          title={dmp.title}
-                          value={dmp.draftId}
-                           onClick={() => handleQuickViewOpen(dmp.draftId)}
-                        >
-                          {truncateText(dmp.title, 50)}
-                        </a>
-
-                        <a
-                          href="#"
-                          class="preview-button"
-                          aria-label="Open plan preview"
-                          onClick={() => handleQuickViewOpen(dmp.draftId)}
-                        >
-                          <svg
-                            xmlns="http://www.w3.org/2000/svg"
-                            height="18"
-                            style={{ top: "3px", position: "relative" }}
-                            viewBox="0 -960 960 960"
-                            width="18"
-                          >
-                            <path d="M433-344v-272L297-480l136 136ZM180-120q-24.75 0-42.375-17.625T120-180v-600q0-24.75 17.625-42.375T180-840h600q24.75 0 42.375 17.625T840-780v600q0 24.75-17.625 42.375T780-120H180Zm453-60h147v-600H633v600Zm-60 0v-600H180v600h393Zm60 0h147-147Z" />
-                          </svg>
-                          <span className="screen-reader-text">
-                            Open Plan Preview
-                          </span>
-                        </a>
-*/}
-                            <div className="d-block table-data-pi">
-                              {dmp.contributors
-                                ? dmp.contributors.items.map((item, index) => (
-                                  <Fragment key={index}>
-                                    {item.roles &&
-                                      item.roles.filter(str => str.includes("investigation")) && (
-                                        <span>
-                                          PI: {truncateText(item.name, 80)}
-                                        </span>
-                                      )}
-                                  </Fragment>
-                                ))
-                                : ""}
-
-                              {dmp.id && dmp.id == "XXX" && (
-                                <span className={"action-required-text"}>
-                                  X works need verification
+
+                          {
+                            <a
+                              href="#"
+                              className="preview-button"
+                              aria-label="Open plan preview"
```

Comment:
Do we need this to be more specific? My understanding was that screen readers gather links together so the user will have multiple 'Open plan preview' links.
Maybe something like `aria-label="Preview plan {truncateText(dmp.title)}"`